Failure to build Hello World example from tutorial

I'm following the tutorial instructions for the ISOBUS Hello World, and I get the following error when calling `cmake --build build`:
```
[ 95%] Built target HardwareIntegration
Scanning dependencies of target isobus_hello_world
[ 97%] Building CXX object CMakeFiles/isobus_hello_world.dir/main.cpp.o
In file included from /home/teo/tms/agisostack/AgIsoStack-plus-plus/isobus/include/isobus/isobus/can_message.hpp:14,
                 from /home/teo/tms/agisostack/AgIsoStack-plus-plus/isobus/include/isobus/isobus/can_callbacks.hpp:14,
                 from /home/teo/tms/agisostack/AgIsoStack-plus-plus/isobus/include/isobus/isobus/can_network_manager.hpp:17,
                 from /home/teo/tms/agisostack/main.cpp:1:
/home/teo/tms/agisostack/AgIsoStack-plus-plus/isobus/include/isobus/isobus/can_control_function.hpp:15:10: fatal error: isobus/utility/
thread_synchronization.hpp: No such file or directory
   15 | #include "isobus/utility/thread_synchronization.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/isobus_hello_world.dir/build.make:63: CMakeFiles/isobus_hello_world.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:158: CMakeFiles/isobus_hello_world.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
```
I've installed AgIsoStack++ according to the instructions:
```
git submodule add https://github.com/Open-Agriculture/AgIsoStack-plus-plus.git
git submodule update --init --recursive
```
And I've successfully built and ran all tests.
I'm running Ubuntu 20.04, and the CXX compiler identification is GNU 9.4.0

I'm a proper noob with C++ so I don't know where to start with this. Let me know if you need any further info to reproduce the issue

--- a/sphinx/source/Tutorials/The ISOBUS Hello World.rst
+++ b/sphinx/source/Tutorials/The ISOBUS Hello World.rst
@@ -59,7 +59,8 @@ Check out the :doc:`concepts section <../Concepts>`, if you need to brush up on 
 
 Let's create a `NAME <https://delgrossoengineering.com/isobus-docs/classisobus_1_1NAME.html>`_ in our program.
 
-NOTE: Everything that is part of the stack is in the namespace `"isobus" <https://delgrossoengineering.com/isobus-docs/namespaceisobus.html>`_!
+.. note::
+   Everything that is part of the stack is in the namespace `"isobus" <https://delgrossoengineering.com/isobus-docs/namespaceisobus.html>`_!
 
 .. code-block:: c++
 
@@ -148,7 +149,7 @@ In this example, I'll use a shared_ptr to store my InternalControlFunction, but 
     return 0;
    }
 
-In this example, I created my control function with a default address of 0x1C, and assigned it to CAN channel 0.
+In this example, I created my control function with a preferred address of 0x1C, and assigned it to CAN channel 0.
 
 Now, we've got a little problem... What is CAN channel 0?
 
@@ -444,7 +445,7 @@ Add the following to a new file called CMakeLists.txt:
    
    add_executable(isobus_hello_world main.cpp)
    
-   target_link_libraries(isobus_hello_world PRIVATE isobus::Isobus isobus::HardwareIntegration Threads::Threads)
+   target_link_libraries(isobus_hello_world PRIVATE isobus::Isobus isobus::HardwareIntegration isobus::Utility Threads::Threads)
 
 Save and close the file.
 

